README.md

```diff
@@ -104,7 +104,7 @@ that you have already set in your shell.
 ```
 . sts.sh <ACCOUNT> <USER NAME> <CODE>
 ```
-
+If you wish to deploy the enclave in debug mode, use the "--debug-mode" flag. You can then log into the instance and view the enclave console.
```

Comment:
While we're at it can we add instructions on how to do this?

Reply:
I can do that in the follow up console command PR!